help/en/releasenotes/current-draft-note.shtml

```diff
@@ -412,7 +412,8 @@
     <h3>LogixNG</h3>
         <a id="LogixNG" name="LogixNG"></a>
         <ul>
-          <li></li>
+          <li>Pressing the F8 key when the spash screen is visible disables all the LogixNGs.</li>
+          <li>Pressing the F9 key when the spash screen is visible prevents LogixNGs to be started.</li>
```

Comment:
Maybe change `LogixNGs to be started` to `LogixNG activation`.

Reply:
> At around line 754 in Apps.java, the following text should be a bundle message and changed.

Fixed.

> Maybe change LogixNGs to be started to LogixNG activation.

Fixed.